### PR TITLE
Cb 346 filtering

### PR DESCRIPTION
**Pull request by pixie79**
Add additional filtering options with tests

gt, gte, lt, lte

### CODE REVIEW

File: app/db/utils.js [hunk 1]
```diff
@@ -20,23 +20,25 @@ function queryFilterDecode(queryParams) {
...
+    } else if (filter === 'gte' && !isNull) {
+      filter = '>=';
+    } else if (filter === 'lt' && !isNull) {
+      filter = '<';
+    } else if (filter === 'lte' && !isNull) {
+      filter = '<=';
+    } else if (filter === 'eq' && !isNull) {
       filter = '=';
     } else if (filter === 'eq' && isNull) {
-      // 'validfrom IS NULL'
```
Comment (p3dr0migue1) [inline app/db/utils.js]: Can you please put these comments back it back please as they are helpful for devs

File: app/db/utils.js [hunk 2]
```diff
@@ -20,11 +20,19 @@ function queryFilterDecode(queryParams) {
...
+    } else if (filter === 'lt' && !isNull) {
+      filter = '<';
+    } else if (filter === 'lte' && !isNull) {
+      filter = '<=';
+    } else if (filter === 'eq' && !isNull) {
+      // 'validfrom IS NULL'
       filter = '=';
     } else if (filter === 'eq' && isNull) {
-      // 'validfrom IS NULL'
+      // 'continent != \'Asia\''
```
Comment (p3dr0migue1) [inline app/db/utils.js]: I think here is `// 'continent === \'Asia\'` Mark, the others are fine

File: app/db/utils.js (hunk 1 above)
Comment (pixie79) [reply app/db/utils.js]: done

File: app/db/utils.js (hunk 2 above)
Comment (pixie79) [reply app/db/utils.js]: yup i noticed that on my push but git froze  i think we are correct now

**Comment by p3dr0migue1** [comment]
:+1:

**Pull request merged**